superset/assets/package.json

```diff
@@ -88,6 +88,7 @@
     "react-addons-shallow-compare": "^15.4.2",
     "react-alert": "^2.3.0",
     "react-bootstrap": "^0.31.5",
+    "react-bootstrap-datetimepicker": "0.0.22",
```

Comment:
just noticed this component is deprecated, should we switch to the non-deprecated package?

https://github.com/YouCanBookMe/react-datetime

Reply:
Let me take a look at how much effort is needed to use that one... I remember testing it when I was looking for calendar components, and it didn't work for some cases but TBH I don't remember the details.